mmcv/utils/seed.py

```diff
@@ -21,3 +25,51 @@ def worker_init_fn(worker_id: int, num_workers: int, rank: int, seed: int):
     np.random.seed(worker_seed)
     random.seed(worker_seed)
     torch.manual_seed(worker_seed)
+
+
+def init_random_seed(seed=None, device='cuda'):
+    """Initialize random seed.
+    If the seed is not set, the seed will be automatically randomized,
+    and then broadcast to all processes to prevent some potential bugs.
+    Args:
+        seed (int, Optional): The seed. Default to None.
+        device (str): The device where the seed will be put on.
+            Default to 'cuda'.
+    Returns:
+        int: Seed to be used.
```

Comment:
```suggestion
    """Initialize random seed.

    If the seed is not set, the seed will be automatically randomized,
    and then broadcast to all processes to prevent some potential bugs.

    Args:
        seed (int, optional): The seed. Default to None.
        device (str): The device where the seed will be put on.
            Defaults to 'cuda'.

    Returns:
        int: Seed to be used.
```